go.mod

```diff
@@ -53,7 +53,7 @@ require (
 	github.com/tomnomnom/linkheader v0.0.0-20180905144013-02ca5825eb80 // indirect
 	github.com/vektra/mockery v1.1.2
 	github.com/voxelbrain/goptions v0.0.0-20180630082107-58cddc247ea2 // indirect
-	github.com/weaveworks/goformation/v4 v4.10.2-0.20210512074147-4fd9b878d08c
+	github.com/weaveworks/goformation/v4 v4.10.2-0.20210512152024-6bd11e940025
```

Comment:
is there a goformation PR we need to merge before this PR?

Reply:
yes! https://github.com/weaveworks/goformation/pull/28